scalability_and_performance/using-topology-manager.adoc

```diff
@@ -0,0 +1,25 @@
+[id="using-topology-manager"]
+= Using Topology Manager
+include::modules/common-attributes.adoc[]
+:context: using-topology-manager
+
+toc::[]
+
+Topology Manager is a Kubelet component that collects hints from CPU Manager
+and Device Manager to align pod CPU and device resources on the same non-uniform memory access (NUMA) node.
+
+Topology Manager uses topology information from collected hints to decide if
+the pod can be accepted or rejected from the node, based on the configured Topology
+Manager policy and Pod resources requested.
+
+Topology Manager is useful for workloads that desire to use hardware accelerators
+to support latency-critical execution and high throughput parallel computation.
+
+[NOTE]
+====
+Topology Manager is an alpha feature in {product-title}.
+====
+
+include::modules/setting-up-topology-manager.adoc[leveloffset=+1]
```

Comment:
@vpickard Note I rearranged the order of topics. If you think the information about policies should come before setting it up, I can change it back. Otherwise, I thought this approach made more sense.

Reply:
@lamek OK, this is fine with me. Thanks.